cypress/integration/search_page_spec.js

```diff
@@ -0,0 +1,207 @@
+const sortSelectBox = '[data-testid=sort-select]'
```

Comment:
Could we move the routes and _maybe_ the selectors to different files?

Reply:
yes

Reply:
Might also be worth namespacing these as well and doing something like
`
export const SearchPage = {
    sortSelectBox: [data-testid=sort-select]',
    paginator = '[data-testid=paginator]'
}
`

Reply:
good idea